addons/tooltips/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Tooltips",
-  "description": "Adds tooltips for projects, users, studios, forum topics, and forum posts. Uses the Scratch API and ScratchDB.",
+  "description": "Adds informational tooltips for links of projects, users, studios, forum topics, and forum posts. Uses the Scratch API and ScratchDB.",
```

Comment:
Shouldn't "links of" be "links to"?

Reply:
`Adds informational tooltips on links to projects, users, studios, forum topics, and forum posts. Uses the Scratch API and ScratchDB for informaton.`

Reply:
They're links to projects because they can be used to go to a project. Similarly, a file path is a path to a file, not of a file.

Reply:
To be honest I would intepret it as path of a file, because it is a path is a noun, with file as an adjective that explains the noun. It's like air conditioner (conditioner of air), hair shampoo (shampoo of hair), etc.

Anyways, I have changed it to "links to projects."

Reply:
i never call shampoo "hair shampoo"

Reply:
Yeah, maybe a bad example, maybe something like "dog shampoo" would be better. Shampoo of dog(s), although shampoo for dogs also also adequate.